galite-core/src/main/kotlin/org/kopi/galite/ui/vaadin/notif/ErrorMessageDetails.kt

```diff
@@ -59,6 +66,7 @@ class VErrorMessagePopup : Popup() {
   //---------------------------------------------------
   // DATA MEMBERS
   //---------------------------------------------------
+  private var details = Details()
```

Comment:
can we add class : 
`class MoreDetails (summary: String, content: Component): Details(summary, content), Focusable<MoreDetails>`
then in this class : 
`private lateinit var details: MoreDetails`
&
add this code in init 
 ```
if (locale == "fr_FR") {
      details = MoreDetails("Afficher les détails",
                        scroller)
    } else if (locale == "en_GB") {
      details = MoreDetails("Show details",
                        scroller)
    }
    details.addThemeVariants(DetailsVariant.REVERSE, DetailsVariant.FILLED)
    add(details)
```